package io.github.VyctorHugoCorreia.exception;

public class ProdutoNaoEncontradoException extends RuntimeException {

    public ProdutoNaoEncontradoException() {
        super("Produto n√£o encontrado.");
    }

}
